packages/appstore/src/components/wallet-action-button/wallet-action-button.scss

```diff
@@ -0,0 +1,23 @@
+.wallet-action-button {
+    background-color: #fff;
```

Comment:
Please change all hex color codes to ones that are in `themes.scss` in the shared package.

Reply:
done :heavy_check_mark: